src/line_ending.rs

```diff
@@ -0,0 +1,117 @@
+//! Line ending detection and conversion.
+
+use std::fmt::{Debug, Formatter};
+use std::str::FromStr;
+
+/// Supported line endings. Like in the Rust's standard library, two
```

Comment:
This should be

> Like in the Rust standard library

or

> Line in Rust's standard library

I prefer the first form without the apostrophe since it seems simpler to a non-native speaker like me :-)